frontend/test/metabase/support/dashboard_data_permissions.cy.spec.js

```diff
@@ -0,0 +1,55 @@
+import {
+  signIn,
+  restore,
+  popover,
+  selectDashboardFilter,
+} from "__support__/cypress";
+
+function filterDashboard(suggests = true) {
+  cy.visit("/dashboard/1");
+  cy.contains("Orders");
+
+  // We should get a suggested response and be able to click it if we're an admin
+  if (suggests) {
+    cy.contains("Category").type("Aero");
+    cy.contains("Aerodynamic").click();
+  } else {
+    cy.contains("Category").type("Aerodynamic Bronze Hat");
+  }
+  cy.contains("Add filter").click();
+  cy.contains("Rows 1-1 of 96");
```

Comment:
I already ran into problems with a similar assertion. The number of **shown** rows ("1-1") directly depends on the size of the card. What we're interested in is the total number of rows ("96").

I'll update this to use Regex.